types/mini-css-extract-plugin/index.d.ts

```diff
@@ -76,6 +76,13 @@ declare namespace MiniCssExtractPlugin {
          * @default 'text/css'
          */
         linkType?: string | false | 'text/css' | undefined;
+        /**
+         * An option to avoid the runtime generation. Assets are still extracted and can be used for a custom loading methods.
+         * For example, you can use [assets-webpack-plugin](https://github.com/ztoben/assets-webpack-plugin) to retreive them
+         * then use your own runtime code to download assets when needed.
+         * @default false
```

Comment:
```suggestion
         * @default true
```

Reply:
sorry, I've misread the changes, amended